### DISCUSSION TITLE
Add 'version' section to GitHub bug report template

### DISCUSSION THREAD

**Discussion by Marcono1234** [ascopes/java-compiler-testing#719] [General]
Hello,
what do you think about adding a section for the JCT version the user is using to the GitHub bug report template? (`.github/ISSUE_TEMPLATE/bug_report.md`)

That might make reproducing easier, especially if you revisit an issue after some time, or if the user is not using the latest version.

**Comment by ascopes** [comment]
Sounds like a plan. Do you want to raise a PR? If not, I can try and sort it out when I get time this week at some point!

**Reply by Marcono1234** [reply]
Thanks for making that change in #723!
